modules/theme/src/themes/default/modules/accordion.overrides

```diff
@@ -2,6 +2,33 @@
       Accordion Overrides
 *******************************/
 
+.ui.accordion {
+    .title {
+        .chevron {
+             transition: transform .1s ease-in-out;
+         }
+         &.active {
+             .chevron {
+                 transform: rotate(90deg);
+             }
+         }
+    }
+}
+
+.ui.accordion {
+    .title {
+        .disabled {
```

Comment:
This style will be applied to the elements with `.disabled` classname in all the accordions.
Shall we scope this to the specific accordion?

Reply:
I couldn't find a prop in React Accordion to disable. Can't we use this for all accordions?